src/clasp_facade.cpp

```diff
@@ -696,13 +725,17 @@ ClaspFacade::Statistics::ClingoView::ClingoView(const ClaspFacade& f) {
 	keys_.add("problem", problem_.toStats());
 	keys_.add("solving", solving_.toStats());
 	keys_.add("summary", summary_.toStats());
+
 	if (f.incremental()) {
 		accu_ = new Accu();
 		accu_->step.bind(*f.accu_.get());
 	}
 	setRoot(keys_.toStats());
 }
 void ClaspFacade::Statistics::ClingoView::update(const ClaspFacade::Statistics& stats) {
+	if (stats.userStats_.get() && !stats.userStats_->empty()) {
+		keys_.add("userdefined", stats.userStats_->toStats());
```

Comment:
Is there a reason why `userdefined` is spelled without a hyphen here? I know that comment is a bit picky, but it might be inconvenient to make such things consistent at a later point in time.

Reply:
For consistency, this should be `user_defined`. Or alternatively, `external` to indicate that these are not maintained by clasp.

Reply:
Okay. I was just referring to the fact that user-defined is a compound adjective and should be written with a hyphen, but if hyphens are not allowed within keys, that’s just how it is :slightly_smiling_face:.

Reply:
Sorry, I forgot to add the "Yep, you are right". I just wanted to express that keys typically use underscores (and unfortunately sometimes camel-case) instead of hyphens.

Reply:
I totally understand, thanks for the info … and sorry about the fuss and my pedantry anyway :smile:!